_bleio/scan_entry.py

```diff
@@ -69,7 +69,9 @@ def _from_bleak(cls, device: BLEDevice) -> "ScanEntry":
         )
 
     def matches(
-        self, prefixes: bytes, all: bool = True  # pylint: disable=redefined-builtin
+        self,
+        prefixes: bytes,
+        match_all: bool = True,  # pylint: disable=redefined-builtin
     ) -> bool:
```

Comment:
Should be able to drop `# pylint: disable=redefined-builtin`, because we are no longer using `all`. Will need another black format after that.
```suggestion
        self,
        prefixes: bytes,
        match_all: bool = True,  # pylint: disable=redefined-builtin
    ) -> bool:
```

Reply:
Oh good catch, I didn't even notice!